index.html

```diff
@@ -203,6 +203,45 @@ <h3><span>&#9679;</span>Skills</h3>
         </div>
       </div>
     </section>
+    <section id="contact">
+      <div class="contact-text">
+        <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+      </div>
+      <form action="https://formspree.io/f/xpznwejk" class="form" method="post">
+        <fieldset>
+          <div class="allNames">
+            <div class="input-group allNames1">
+            <label>
+            <input type="text" id="firstName" class="fName" name="firstName" placeholder="First name" required> 
+          </label>
+        </div>
+          <div class="input-group allNames1">
+            <label>
+            <input type="text" id="lastName" class="lName" name="lastName" placeholder="Last name" required>
+          </label>
+          </div>
+          </div>
+          <div class="input-group fullN">
+            <label>
+            <input type="text" id="fullName" name="fullName" placeholder="Full name" required>
+          </label>
+          </div>
+          <div class="input-group">
+            <label>
+            <input type="email" id="email" name="email" placeholder="name@gmail.com" required>
+          </label>
+          </div>
+          <div class="input-group">
+            <label>
+            <textarea name="msg" id="msg" cols="30" rows="10" minlength="30" placeholder="Enter text here" required></textarea>
+          </label>
+          </div>
+          <div class="submit">
+            <button type='submit'>Get in touch</button>
+          </div>
```

Comment:
- [ ] Please check the functionality of your app. I have tried the contact form to submit using the button it's not working, and the validation for the input ~name: fullName. Kindly fix this for reference[ check this](https://www.the-art-of-web.com/html/html5-form-validation/)